src/features/FreeRooms/FreeRooms.ts

```diff
@@ -0,0 +1,335 @@
+import ROOMS from './Rooms';
```

Comment:
I would say just to be safe, we would extract the rooms based on what the website presents to us visually (as opposed to hard-coding the room values). This will be easier to do with a web-scraping or HTML-parsing library.